x-pack/plugins/maps/public/classes/styles/vector/properties/components/breaked_legend.js

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
Can you move this file to `public/classes/styles/vector/components/legend` folder? There should not be a components folder under `properties`. This makes the imports confusing trying to figure out which components folder to use